dp0.tex

```diff
@@ -36,7 +36,7 @@ \subsection{Elements of Data Preview 0.1}
 
 \subsubsection {Dataset choice considerations} \label{sec:dataset}
 
-For DP0.1 we are moving ahead with DESC DR6 as the baseline. We may still load HSC at some point.
+For DP0.1 we are moving ahead with DESC DR6 WFD as the baseline. We may still load HSC at some point.
```

Comment:
At some point _in DP0.1_ or later on, in the DP0.2 time frame?

Reply:
No within the DP0.2 time frame, but maybe later.  Currently there isn't a really plan loading HSC data, as far as I know.

Reply:
Then I would say "We may still load Rubin-processed HSC data at some later point in the DP0 era."

Reply:
Re-reading it, I find mentioning HSC here confusing.  So I'm removing this sentence, and referring readers to the Appendix where some HSC data consideration is discussed.